summerofcode/2023.md

```diff
@@ -26,12 +26,36 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ## Ideas
 
+- [Jaeger](#jaeger)
+  - [Implement ClickHouse support](#implement-clickhouse-support)
 - [Knative](#knative)
   - [Multiple Knative Eventing control planes](#multiple-knative-eventing-control-planes)
   - [Eventing Sender Identity](#eventing-sender-identity)
   - [NetworkPolicy support for Knative Serving](#networkpolicy-support-for-knative-serving)
   - [Improving Observability in Knative Eventing](#improving-observability-in-knative-eventing)
 
+### Jaeger
+
+Jaeger (https://jaegertracing.io) is a popular platform for distributed tracing.
+
+#### Implement ClickHouse support
+
+- Description: ClickHouse is a powerful columnar database, utilized in many commercial and some open source monitoring products. Jaeger currently has a community-provided (unofficial) plugin that implements Jaeger storage backend on top of ClickHouse. This project is to build first-class support for ClickHouse in Jaeger as a core functionality.
+- Expected outcomes:
+  - Design table schema, document trade-offs
+  - Implement ClickHouse support as core storage backend
+  - Address schema creation problem / tooling
+  - Add relevant documentation to the Jaeger website
+- Stretch goals:
+  - Use Kafka Connector for ClickHouse for bulk inserts
+  - Support Monitoring tab in Jaeger directly from ClickHouse
+- Recommended Skills: Go, SQL
+- Mentor(s): Yuri Shkuro, @yurishkuro
```

Comment:
I should have mentioned sooner, @yurishkuro could you update this with your email address? If you don't want to put it here, you can also DM me on the CNCF slack.

Reply:
github@ysh.us